webpack.dev.config.babel.js

```diff
@@ -71,6 +71,7 @@ export default Object.assign({}, webpackConfig, {
       'react-dom/umd/react-dom.development.js',
     ),
     new webpack.NormalModuleReplacementPlugin(/^redux$/, 'redux/dist/redux.js'),
+    new webpack.NamedModulesPlugin(),
```

Comment:
D'oh. Good find. It looks like this will be the default for development in [webpack 4](https://github.com/webpack/webpack/issues/3476#issuecomment-368300958).